terraform/environment/db_cluster_applications.tf

```diff
@@ -1,70 +1,70 @@
-# resource "aws_rds_cluster" "applications" {
-#   cluster_identifier     = "applications-${local.environment}"
-#   vpc_security_group_ids = [aws_security_group.applications_rds_cluster.id]
-#   db_subnet_group_name   = "${aws_db_subnet_group.applications_rds_cluster.name}"
+resource "aws_rds_cluster" "applications" {
+  cluster_identifier     = "applications-${local.environment}"
+  vpc_security_group_ids = [aws_security_group.applications_rds_cluster.id]
+  db_subnet_group_name   = aws_db_subnet_group.applications_rds_cluster.name
 
-#   engine                          = "aurora-postgresql"
-#   engine_mode                     = "serverless"
-#   engine_version                  = "10.7"
-#   db_cluster_parameter_group_name = "default.aurora-postgresql10"
+  engine                          = "aurora-postgresql"
+  engine_version                  = "9.6.11"
+  db_cluster_parameter_group_name = "default.aurora-postgresql9.6"
 
-#   backtrack_window  = 0
-#   apply_immediately = true
-#   storage_encrypted = true
+  backtrack_window  = 0
+  apply_immediately = true
+  storage_encrypted = true
 
-#   master_username = local.rds_master_username
-#   master_password = data.aws_secretsmanager_secret_version.postgres_password.secret_string
+  master_username = local.rds_master_username
```

Comment:
the username here is defined in locals.tf. Does it match what the migrated prood db requires?